packages/react-adminer/src/components/CellEditInput.tsx

```diff
@@ -37,7 +37,14 @@ const CellEditInput: React.FC<Props> = ({
 		const payload = { [propertyName]: value };
 		const fn = async (): Promise<void> => {
 			try {
-				await dataProvider.update(entityName, payload, config, { where: { id } });
+				await dataProvider.update(
+					entityName,
+					fixFormat(config.fields.find(e => e.name === propertyName)?.type, value, propertyName, payload),
```

Comment:
Jeste bych to "vyhezcil", posli cely ten config do te funkce a tam si to az najdi.

Jinak takova "find" funkce se tady urcite vyskytuje vicekrat, tak by bylo fajn "ji dat na jedno misto" - utils - config (neni tam uz?)

Treba `getConfigByEntityName` a ta ti ten find udela - jde o to to zjednodusit, mit na jednom miste, idealne z nazvu funkce vidis co to dela

Reply:
Takto nějak?